webtool/test_correct.py

```
@@ -2077,3 +2082,110 @@ def remove_mit_marke(pfad, *a, **kw):
+def test_glossar_sperrt_alle_gelesenen_aufnahmen(monkeypatch, tmp_path, capsys):
+    """Waehrend das Glossar liest, ist JEDE gelesene Aufnahme gesperrt — auch die fremde (#450).
+
+    `_glossary` liest die `.raw.txt` des ganzen Korpus; `llm._with_files` haelt dabei je einen
+    Griff offen. Ohne Marken ist `active_bases` leer (`[scope]` fuellt `bases`), `DELETE
+    …/files/{base}` kommt mit `active_only=True` durch und stirbt in `app._datei_weg` am
+    `PermissionError` — halb geloescht. Am echten Pfad gemessen: vorher 500 + Waise, jetzt 409.
+    """
+    _glossar_projekt(monkeypatch, tmp_path)
+    marke = "marke: _ask_llm liest die .raw.txt"
+
+    def ask_llm_mit_marke(prompt, inputs, output):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `ask_llm_mit_marke`

Add return type annotation: `None`

(ANN202)